app/build.gradle

```diff
@@ -13,13 +13,33 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    signingConfigs {
+        debug {
+            // keytool -genkeypair -v -keystore my-debug-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+            // ./gradlew assemble -Dpassword=123456
+            storeFile file('my-debug-key.keystore')
+            storePassword System.getProperty('password')
+            keyAlias 'my-key-alias'
+            keyPassword System.getProperty('password')
+        }
+        release {
+            // keytool -genkeypair -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+            // ./gradlew assemble -Dpassword=123456
+            storeFile file('my-release-key.keystore')
```

Comment:
@xinminsu Can you also make the `storeFile` a command line parameter? This is because we already have a keystore file on the build server that we will be using.